### PR TITLE
Add more links in the RHN

### PR DESCRIPTION
**Pull request by mountiny**
Adding more links from the About page to Search page and back to settings to be able to test the navigation behaviour in that scenario.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -105,7 +105,17 @@ function AboutScreen({navigation}) {
             onPress={() => navigation.push('Chat', {id: 1})}
           >
             <Text style={{color: 'blue', fontSize: 18, marginBottom: 10}}>Link to another chat</Text>
-          </Pressable>
+        </Pressable>
+        <Pressable
+            onPress={() => navigation.push('Settings')}
+          >
+            <Text style={{color: 'blue', fontSize: 18, marginBottom: 10}}>Link to different part of settings</Text>
```
Comment (marcaaron) [inline App.js]: I'm not sure if it's a use case we have, but if we did I could see an argument that a link to "Settings" would actually go "up/back" and maybe not push a new settings stack route.

File: App.js [hunk 2]
```diff
@@ -105,7 +105,17 @@ function AboutScreen({navigation}) {
...
+        </Pressable>
+        <Pressable
+            onPress={() => navigation.push('Search')}
+          >
+            <Text style={{color: 'blue', fontSize: 18, marginBottom: 10}}>Link to Search page</Text>
+        </Pressable>
```
Comment (marcaaron) [inline App.js:113]: Similar to previous question, but wondering if we need to spend much time thinking about cases we do not currently (and may never have to) support.

File: App.js (hunk 1 above)
Comment (mountiny) [reply App.js]: My reasoning behind this is to add more flows then less to test the POC as much as possible, but I agree we would try to handle this by go-back in real app

File: App.js (hunk 2 above)
Comment (mountiny) [reply App.js:113]: Well here I am just thinking if we wont ever have a case we would switch between the RHN screens, there is Workspace, Settings, Search there could be more coming over time so making sure the navigation works fine between these if I think useful

**Comment by mountiny** [comment]
@marcaaron Updated.

File: App.js (hunk 1 above)
Comment (marcaaron) [reply App.js]: Cool. I am just cautious about spending much time on things that we may never need.

So do you think that it makes sense to pop the pages off as if we were navigating back? 
Or push another settings stack onto the main stack?

Though I would prefer not to have to even answer that question and just say - we won't ever try to do this. 😂

File: App.js (hunk 2 above)
Comment (marcaaron) [reply App.js:113]: So basically the question is:

What should happen if/when we are deep linking from one part of nested navigator to another part of a nested navigator?

Maybe the simplest thing is to allow a "push" to happen. I think this type of navigation might be pretty rare, but if you e.g. followed one link from `[LHN, [Settings, About]]` to `[LHN, [Settings, Workspace]]` then the only way to preserve the correct "BACK" behavior is to add a new stack.

**Review by marcaaron — APPROVED** [review]

File: App.js (hunk 1 above)
Comment (mountiny) [reply App.js]: > Though I would prefer not to have to even answer that question and just say - we won't ever try to do this. 😂

Copy that, wont attempt to answer 😄

File: App.js (hunk 2 above)
Comment (mountiny) [reply App.js:113]: > Maybe the simplest thing is to allow a "push" to happen. I think this type of navigation might be pretty rare, but if you e.g. followed one link from [LHN, [Settings, About]] to [LHN, [Settings, Workspace]] then the only way to preserve the correct "BACK" behaviour is to add a new stack.

Let me explore this, we should try to navigate from About page to Workspace or something like this I assume

**Pull request merged**